### PR TITLE
Add CI.md with Github Action details

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

This PR adds CI.md that explains the Github Actions that are added to this repository.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: CI.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+##  cloud.common Collection
+
+GitHub Actions are used to run the Continuous Integration for ansible-collections/cloud.common collection. The workflows used for the CI can be found [here](https://github.com/ansible-collections/cloud.common/tree/main/.github/workflows). These workflows include jobs to run the unit tests, sanity tests, linters and changelog check. The following table lists the python and ansible versions against which these jobs are run.
+
+| Jobs | Description | Python Versions | Ansible Versions |
+| ------ |-------| ------ | -----------|
+| changelog |Checks for the presence of Changelog fragments | 3.9 | devel |
+| Linters | Runs `black` and `flake8` on plugins and tests | 3.9 | devel |
+| Unit tests | Executes the unit test cases | 3.9, 3.10 | Stable-2.12+ |
+| Sanity | Runs absible sanity checks | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
```
Comment (gravesm) [inline CI.md:12]:
```suggestion
| Sanity | Runs ansible sanity checks | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
```

**Comment by alinabuzachis** [comment]
@GomathiselviS @gravesm 
I'm not sure, but this collection requires python 3.6 or higher. If we still have this requirement, shouldn't we also test against 3.6 and 3.7?

**Comment by GomathiselviS** [comment]
> @GomathiselviS @gravesm I'm not sure, but this collection requires python 3.6 or higher. If we still have this requirement, shouldn't we also test against 3.6 and 3.7?

From https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html, I see that from ansible 2.12 there is no support for py 3.6 and 3.7 on the controller node. Please correct me if I am wrong.

**Comment by gravesm** [comment]
> From https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html, I see that from ansible 2.12 there is no support for py 3.6 and 3.7 on the controller node. Please correct me if I am wrong.

The collection supports earlier versions of python for module execution, though, so we should be testing those as well.

**Comment by GomathiselviS** [comment]
> > From https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html, I see that from ansible 2.12 there is no support for py 3.6 and 3.7 on the controller node. Please correct me if I am wrong.
> 
> The collection supports earlier versions of python for module execution, though, so we should be testing those as well.
Support added in https://github.com/ansible-collections/cloud.common/pull/127

**Review by gravesm — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**